auth/api/iam/api.go

```diff
@@ -59,6 +79,17 @@ func (r Wrapper) Routes(router core.EchoRouter) {
 			return audit.StrictMiddleware(f, vcr.ModuleName+"/v2", operationID)
 		},
 	}))
+	// The following handler is of the OpenID4VCI wallet which is called by the holder (wallet owner)
+	// when accepting an OpenID4VP authorization request.
+	router.POST("/iam/:did/openid4vp_authz_accept", r.handlePresentationRequestAccept, audit.Middleware(vcr.ModuleName+"/v2", "openid4vp_authz_accept"))
```

Comment:
add comment that these need to be added to client redirect_uris (part of security model to prevent request forgery after obtaining an authorization code)